pkg/controller/node/node_controller.go

```diff
@@ -503,7 +515,11 @@ func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result,
 		var clusterTag = model.Tag{Scope: &clusterScope, Tag: &cluster}
 		segmentPort.Tags = append(segmentPort.Tags, clusterTag)
 	}
-	_, err = portsClient.Update(segmentId, *segmentPort.Id, *segmentPort)
```

Comment:
Sorry I did not know we still need portsClient here

Reply:
Yes and I found the Update methods from infra_segment and tier1_segment are different so in the latest change I split the clients to infraPortsClient and tier1PortsClient